README.md

```diff
@@ -1,2 +1,35 @@
-# oras-actions
-GitHub Actions for accessing artifacts in OCI-compliant registries
+# setup-oras
+
+![ORAS](https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png)
+
+Setup [ORAS CLI](https://github.com/oras-project/oras) on GitHub Actions runners.
+
+## Usage
+
+Setup the `oras` CLI:
+
+```yaml
+steps:
+  - uses: oras-project/setup-oras@main
+  - run: oras version
+```
+
+Install a specific version of the `oras` CLI:
+
+```yaml
+steps:
+  - uses: oras-project/setup-oras@main
+    with:
+      version: 0.16.0
```

Comment:
Can we add documentation how does one request specific version of ORAS? Not only 0.16.0

Reply:
This is the documentation to request specific version of oras. I will re-write this basic documentation to a detailed one.